processors/video-processor/index.js

```diff
@@ -11,14 +11,15 @@ const winston = require('winston');
 
 
 exports.process = function (job, done) {
-    if (!process.env.FFMPEG_PATH) {
+    if (config.ffmpeg_path) {
```

Comment:
This was always set in `process-video-service/service.js`, so the value from the config file was never used. This is just to say, if there is something in the config, use it, otherwise it should fall back to `process-video-service/service-config,json`.